examples/with-react-ga4/README.md

```diff
@@ -1,6 +1,6 @@
 # React-GA example
 
```

Comment:
Could you add a note at the beginning of the README that explains why `react-ga4` is used instead of `react-ga`? Something along the lines of:

```
As of this change, [React-GA](https://github.com/react-ga/react-ga ) uses Universal Analytics which will be deprecated in July 1, 2023. Until this is fixed, this example has been updated to use the `React-GA4` library.
```

Reply:
I should approve your review but I'm relatively new to pull requests so I didn't know about it, sorry